code/modules/spacepods/spacepod.dm

```diff
@@ -127,6 +123,7 @@
 	cargo_hold.storage_slots = 0	//You need to install cargo modules to use it.
 	cargo_hold.max_w_class = 5		//fit almost anything
 	cargo_hold.max_combined_w_class = 0 //you can optimize your stash with larger items
+	START_PROCESSING(SSobj, src)
```

Comment:
Need to remove the `START_PROCESSING(SSobj, src)` from `/obj/spacepod/emp_act(severity)`

Reply:
Done